src/main/scala/com/datastax/cdm/job/BaseJob.scala

```diff
@@ -55,6 +51,7 @@ abstract class BaseJob[T: ClassTag] extends App {
     sContext = spark.sparkContext
     sc = sContext.getConf
     propertyHelper = PropertyHelper.getInstance(sc);
+    this.partitionFileName = getPartitionFile(propertyHelper);
```

Comment:
Please confirm that the partitionFileName is only read for job executions with the partitions range file.

Reply:
@faizalrub-datastax The jobs (`Migrate` & `DiffData`) will alway look for a partition-range file. By default it looks for  file`./<keyspacename>.<tablename>_partitions.csv`, but you can also use a different file using config/param  `spark.tokenrange.partitionFile`. If it finds this file, it will then use it to migrate/validate only those partition-ranges. However, if no such file is present, it will do the full migration or validation.

Reply:
**Note**: Migrate & DiffData will continue to work as before i.e. perform full migration by default. However if you pass a file with partition-ranges, it will apply migration/validation only on those ranges.